vcpkg/users/versioning-troubleshooting.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Troubleshoot versioning and feature selection issues
+description: Troubleshooting guide for common versioning issues
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 11/30/23
+ms.prod: vcpkg
+ms.topic: troubleshooting-general
+---
+
+# Versioning troubleshooting guide
+
+This guide is intended for users experiencing issues with [versioning](./versioning.md).
```

Comment:
Maybe we should add a section about inspecting the version file fora port, so users know to look for `versions/<p>-/<port>`